<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalToolManager</class>
 <widget class="QDialog" name="ExternalToolManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/images/actions/dialog-close.svg</normaloff>:/images/actions/dialog-close.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QTableView" name="tableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Accept</string>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/images/actions/dialog-ok-apply.svg</normaloff>:/images/actions/dialog-ok-apply.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="itemDown">
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/images/actions/go-down.svg</normaloff>:/images/actions/go-down.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>172</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="itemDel">
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/images/actions/list-remove.svg</normaloff>:/images/actions/list-remove.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="itemAdd">
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/images/actions/list-add.svg</normaloff>:/images/actions/list-add.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="itemUp">
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/images/actions/go-up.svg</normaloff>:/images/actions/go-up.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExternalToolManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ExternalToolManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
